linkerd/retry/src/lib.rs

```diff
@@ -1,21 +1,25 @@
 #![deny(rust_2018_idioms, clippy::disallowed_methods, clippy::disallowed_types)]
 #![forbid(unsafe_code)]
 
-use futures::future;
+use futures::future::{self, Either};
```

Comment:
mildly preferable to take the vendored Either for consistency.

Reply:
for posterity, this was addressed in 9bded3d